### PR TITLE
Remove unused 'artifactory.releasebundle.feature.enabled'

### PR DESCRIPTION
**Pull request by eldada**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated

**What this PR does / why we need it**:
Remove unused 'artifactory.releasebundle.feature.enabled'.

### CODE REVIEW

File: stable/artifactory/templates/artifactory-statefulset.yaml [hunk 1]
```diff
@@ -208,9 +208,6 @@ spec:
           {{- if .other }}
             {{ .other }}
           {{- end }}
-      {{- end }}
```
Comment (danielezer) [inline stable/artifactory/templates/artifactory-statefulset.yaml]: Why did you remove this end?

File: stable/artifactory/templates/artifactory-statefulset.yaml [hunk 2]
```diff
@@ -186,7 +186,7 @@ spec:
               key: master-key
         - name: EXTRA_JAVA_OPTIONS
           value: >
-      {{- with .Values.artifactory.javaOpts }}
+        {{- with .Values.artifactory.javaOpts }}
```
Comment (danielezer) [inline stable/artifactory/templates/artifactory-statefulset.yaml]: Please fix this indentation :)

Comment (eldada) [reply stable/artifactory/templates/artifactory-statefulset.yaml]: This is aligned with what we have in `artifactory-ha`.

Comment (danielezer) [reply stable/artifactory/templates/artifactory-statefulset.yaml]: 👍
